solr/core/src/test/org/apache/solr/store/shared/SharedCoreConcurrencyTest.java

```diff
@@ -595,7 +595,7 @@ private void configureTestSharedConcurrencyControllerForProcess(
       public void recordState(String collectionName, String shardName, String coreName, SharedCoreStage stage) {
         super.recordState(collectionName, shardName, coreName, stage);
         ConcurrentLinkedQueue<String> coreConcurrencyStages = coreConcurrencyStagesMap.computeIfAbsent(coreName, k -> new ConcurrentLinkedQueue<>());
-        coreConcurrencyStages.add(Thread.currentThread().getId() + "." + stage.name());
+        coreConcurrencyStages.add(Thread.currentThread().getName() + "." + stage.name());
```

Comment:
what was the reason for this change?

Reply:
Sorry this is some what irrelevant. Looking through one of test run logs I realized in logging if thread has a name then that is logged e.g. puller threads. This change is only to help make debugging of this test easier.